db/models.js

```diff
@@ -20,5 +20,30 @@ const getArticleData = async (id) => {
   const article = await checkExists('articles', 'article_id', id);
   return article.rows[0]
 };
+const getArticlesData = async () => {
+  const articles = await db.query(
+    "SELECT * FROM articles ORDER BY created_at;"
+  );
+  const comments = await db.query("SELECT * FROM comments;");
```

Comment:
You should keep things consistent. If you are using async await you should do this all the way through.